draft-ietf-quic-transport.md

```diff
@@ -512,25 +511,24 @@ from different versions of QUIC are interpreted.
 
 ## Version Negotiation Packet {#packet-version}
 
-A Version Negotiation packet has long headers with a type value of 0x7F and is
-sent only by servers.  The Version Negotiation packet is a response to a client
-packet that contains a version that is not supported by the server.
+A Version Negotiation packet is inherently not version-specific, and does not
+use the packet headers defined above.  The Version Negotiation packet is a
+response to a client packet that contains a version that is not supported by the
+server, and is only sent by servers.
 
-The connection ID and version fields echo corresponding values from the
-triggering client packet.  This allows clients some assurance that the
-server received the packet and that the Version Negotiation packet was not
-carried in a packet with a spoofed source address.
-
-A Version Negotiation packet is never explicitly acknowledged in an ACK frame by
-a client.  Receiving another Initial packet implicitly acknowledges a Version
-Negotiation packet.
-
-The payload of the Version Negotiation packet is a list of 32-bit versions which
-the server supports, as shown below.
+The layout of a Version Negotiation packet is:
 
 ~~~
  0                   1                   2                   3
  0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
++-+-+-+-+-+-+-+-+
+|1|   Type (7)  |
```

Comment:
Rather than enshrine this as a "Type" field, I think that I'd prefer that this be marked "Unused" instead.